Cargo.toml

```diff
@@ -24,6 +24,7 @@ serde = "0.6.13"
 serde_json = "0.6.0"
 serde_macros = "0.6.14"
 staticfile = "0.1.0"
+unicase = "*"
```

Comment:
Any reason to not pin at unicase = "1.3.0" which is the current latest version in crates.io ?